upgradability/set_code_hash/transparent_voting/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore build artifacts from the local tests sub-crate.
+/target/
+
+# Ignore backup files creates by cargo fmt.
+**/*.rs.bk
```

Comment:
I suggest keeping a single `.gitignore` file in the root directory capturing all such files with globs (like all `target` directories)